contracts/contracts/PoolDeposits.sol

```diff
@@ -204,11 +209,12 @@ contract PoolDeposits {
   /// @param amount the user wants to deposit into the DAOcare pool
   function deposit(uint256 amount)
     external
-    blankUser
```

Comment:
No need to check they are a blank user. The voting contract rather checks they haven't voted in this iteration.